tests/tests_suite_test.go

```diff
@@ -60,31 +60,31 @@ func TestTests(t *testing.T) {
 	if qe_reporters.JunitOutput != "" {
 		junitOutput = qe_reporters.JunitOutput
 	}
-	if config.GinkgoConfig.ParallelTotal > 1 {
-		artifactsPath = filepath.Join(artifactsPath, strconv.Itoa(config.GinkgoConfig.ParallelNode))
-		junitOutput = filepath.Join(flags.ArtifactsDir, fmt.Sprintf("partial.junit.functest.%d.xml", config.GinkgoConfig.ParallelNode))
+
+	suiteConfig, _ := GinkgoConfiguration()
+	if suiteConfig.ParallelTotal > 1 {
+		artifactsPath = filepath.Join(artifactsPath, strconv.Itoa(GinkgoParallelProcess()))
+		junitOutput = filepath.Join(flags.ArtifactsDir, fmt.Sprintf("partial.junit.functest.%d.xml", GinkgoParallelProcess()))
 	}
 
 	outputEnricherReporter := reporter.NewCapturedOutputEnricher(
-		ginkgo_reporters.NewJUnitReporter(junitOutput),
+		v1reporter.NewV1JUnitReporter(junitOutput),
```

Comment:
Do we need the custom junit reporter anymore ?

Reply:
yes, we are using it and not the built-in reporter, which has different XSD scheme
using the new reporter can be done on a follow PR, but needs lot of adjustments,
for the tests, unit-tests, junit-merger, flake finder, prow spyglass and so on
so for now we are using the good old version as a custom reporter (also polarion reporter that is used uses the same)
this PR is too massive already, so this was the most self contained solution i found (tried both and it didn't converge well)

inspired by
https://github.com/gardener/gardener/pull/5313